opentelemetry-stdout/examples/basic.rs

```diff
@@ -64,16 +64,20 @@ fn init_logs() -> opentelemetry_sdk::logs::LoggerProvider {
 
 #[cfg(feature = "trace")]
 fn emit_span() {
-    use opentelemetry::trace::{
-        SpanContext, SpanId, TraceFlags, TraceId, TraceState, TracerProvider,
+    use std::sync::Arc;
+
+    use opentelemetry::{
+        trace::{SpanContext, SpanId, TraceFlags, TraceId, TraceState, TracerProvider},
+        InstrumentationLibrary,
     };
 
-    let tracer = global::tracer_provider()
-        .tracer_builder("stdout-example")
-        .with_version("v1")
-        .with_schema_url("schema_url")
-        .with_attributes([KeyValue::new("scope_key", "scope_value")])
-        .build();
+    let library = Arc::new(
```

Comment:
From a useability standpoint, I think its best to not require Arc here. Cloning InstrumentationLibrary should be cheap anyway.

Reply:
`InstrumentationLibrary` was already cheap to clone, except for the `attributes`. I've put them in a `Cow` so we no longer need to use `Arc`.

Reply:
I noticed that you changed it to `pub attributes: Cow<'static, Vec<KeyValue>>`. How does wrapping the vec with `Cow` help?

Did you mean to change that to cow of static slice instead? `Cow<'static, [KeyValue]>`?

Reply:
Its okay to not optimize for cloning cost. 
Cloning same instrumentation-scope is to be done at max for 2 times (one for each signal), so it is okay!
(Or on the future, we can do Scope backed by a Arc<InnerScope> so clone is super cheap (if we ever need that))

Reply:
I've reverted the `attributes` type, but this raises the question of using `Cow` for the other struct members.